tardis/montecarlo/tests/test_rpacket.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "../src/rpacket.h"
+
+int testing_rpacket_get_nu();
+
+int
+testing_rpacket_get_nu(){
+	rpacket_t * rp = (rpacket_t *) malloc(sizeof(rpacket_t));
+	double value = 10.2;
+	/*rpacket_set_nu(rp, value);
```

Comment:
@wkerzendorf if I uncomment this particular line, the compiler is throwing an error. How do I call the function `rpacket_set_nu` from `rpacket.h` ?